modules/gdscript/language_server/gdscript_language_protocol.h

```diff
@@ -31,33 +31,56 @@
 #ifndef GDSCRIPT_PROTOCAL_SERVER_H
 #define GDSCRIPT_PROTOCAL_SERVER_H
 
+#include "core/io/stream_peer.h"
+#include "core/io/stream_peer_tcp.h"
+#include "core/io/tcp_server.h"
 #include "gdscript_text_document.h"
 #include "gdscript_workspace.h"
 #include "lsp.hpp"
 #include "modules/jsonrpc/jsonrpc.h"
-#include "modules/websocket/websocket_peer.h"
-#include "modules/websocket/websocket_server.h"
+
+#define LSP_MAX_BUFFER_SIZE 4194304
+#define LSP_MAX_CLIENTS 8
 
 class GDScriptLanguageProtocol : public JSONRPC {
 	GDCLASS(GDScriptLanguageProtocol, JSONRPC)
 
+private:
+	class LSPeer : public Reference {
```

Comment:
I'd like to use `struct` instead of `class` as all of the members are public and no constructor need as we can use inline initializing.

Reply:
It's not just a class it's reference, to avoid explicit deallocation, also, having `Vector` and `Ref` as members it is still not a `POD`, it needs an explicit initialization I think.

Reply:
my bad, you only meant to define it as `struct` not changing the inheritance, that make sense.